packages/odata-common/src/selectable/field-builder.ts

```diff
@@ -50,7 +52,10 @@ export type IsSelectableField<FieldOfT extends ConstructorOrField<any>> =
 export type IsOrderableField<EdmT extends EdmTypeShared<'any'>> =
   EdmT extends OrderableEdmType ? true : false;
 
-type EntityTypeFromFieldOf<FieldOfT extends ConstructorOrField<any>> =
+/**
+ * PLEASE REVIEW.
```

Comment:
Helper type to extract the entity from a field so EntityTypeFromFieldOf<EdmTypeField<MyEntity>> is `MyEntity`.